Documentation/installation/k8s-install-migration.rst

```diff
@@ -193,6 +193,34 @@ Preparation
         $ cilium install |CHART_VERSION| --values values-migration.yaml --dry-run-helm-values > values-initial.yaml
         $ cat values-initial.yaml
 
+.. note::
+When you install Cilium, it will create a new network interface called ``cilium_host`` on each node. You may need to configure the old CNI to ensure that it doesn't accidentally use this interface.
```

Comment:
Wording for clarity
```suggestion
When you install Cilium, it creates a new network interface called ``cilium_host`` on each node. Configure the old CNI to ensure that it doesn't accidentally use this interface.
```

Reply:
I am not sure you need to (or can) do this for all types of CNI, but we do know that it applies in the case of Calico. That's why I suggested the wording "You may need to", and I still think it's more accurate that way.

Reply:
If you don't know and we don't know, how will the user know?

Reply:
I think this is maybe related to the [commentary I had](https://github.com/cilium/cilium/pull/27666#discussion_r1304746464): Rather than saying "You may need to", if we say something more like "Ensure that existing plugins are configured to avoid modifying Cilium-managed traffic" then that is a bit more explicit. Later on in the document it'll describe specifically the Calico case so that one will be clear. For other CNIs it may or may not apply, but we make it clear that it's up to the user to make sure that property holds.

Reply:
> Rather than saying "You may need to", if we say something more like "Ensure that existing plugins are configured to avoid modifying Cilium-managed traffic" then that is a bit more explicit.

I think this wording works better, with some modifications. I've updated my review comment accordingly.
```

Reply:
Our comments here may be difficult to reconcile with @joestringer's https://github.com/cilium/cilium/pull/27666#discussion_r1304746464. I'll give a fresh review that attempts to reconcile both; I welcome feedback!